PromiseTests/PromiseAlwaysTests.swift

```diff
@@ -20,9 +20,9 @@ class PromiseAlwaysTests: XCTestCase {
             }
         })
         
-        promise.always({ _ in
+        promise.always {
             expectation?.fulfill()
-        })
```

Comment:
revert these please, the style of this project is to never use trailing block syntax

Reply:
Will do.